### PR TITLE
Update day2 readme

### PR DESCRIPTION
**Pull request by ndufreche**
Just readme content update

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
thanks !

File: README.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
-This branch is the result you get when you finish day 1 of blogbee tutorial (except vendors and project configuration files).
+This branch is the result you get when you finish day 2 of blogbee tutorial (except vendors and project configuration files).
 
 The following installation instructions consider that you have already installed the application requirements such as git, composer, php and nginx or apache2, if not, see [blogbee tutorial day1](https://github.com/backbee/blogbee-docs/blob/master/doc/day1.md)
 
+The result of the integration is available integration folder.
```
Comment (eric-chau) [inline README.md]: The end of this sentence is kinda strange no?

**Comment by eric-chau** [comment]
Too quickly merged :/

File: README.md (hunk 1 above)
Comment (ndufreche) [inline README.md]: I forgot one word I think "in"

Comment (eric-chau) [inline README.md]: I think ``The integration is available into integration folder`` is enough.

**Pull request merged**